target/snitch_cluster/sw/snax/data-reshuffler/Makefile

```diff
@@ -39,9 +39,20 @@ INCDIRS += $(SNRT_DIR)/src/omp
 INCDIRS += $(SNRT_DIR)/api/omp
 INCDIRS += $(SNRT_DIR)/vendor/riscv-opcodes
 INCDIRS += $(ROOT)/target/snitch_cluster/sw/runtime/common
+
+# math.h needed by snRuntime
+
+INCDIRS += $(SNRT_DIR)/../math/arch/riscv64/
+INCDIRS += $(SNRT_DIR)/../math/arch/generic
+INCDIRS += $(SNRT_DIR)/../math/src/include
+INCDIRS += $(SNRT_DIR)/../math/src/internal
+INCDIRS += $(SNRT_DIR)/../math/include/bits
+INCDIRS += $(SNRT_DIR)/../math/include
+
```

Comment:
If these are already in `common.mk` do we have to declare them here too?

Reply:
Yes, this piece of code is repeated too much! Maybe put them into common

Reply:
Let's copy-paste for now and do in a separate PR refactoring all the Makefile libraries 😄